src/CMakeLists.txt

```diff
@@ -130,13 +130,30 @@ if(CMAKE_CXX_COMPILER_ID STREQUAL "MSVC")
   target_compile_definitions(main
     PRIVATE
       NOMINMAX)
+
+  # Dependency from common.h
+  find_library(B64_LIBRARY NAMES b64)
```

Comment:
What will happen if library not found?

Reply:
Can we try to be consistent in our scenarios and make change similar to. 
https://github.com/triton-inference-server/perf_analyzer/commit/0c589f85dbab0b55d709cb8e2baf8fbfc27f7847

Reply:
@KrishnanPrash

Reply:
If it's acceptable, would it be possible to get this in as a follow-up PR?